app/test_main.py

```diff
@@ -1 +1,29 @@
+
 # write your code here
+from unittest import mock
+import pytest
+
+from app.main import can_access_google_page
+
+
+@mock.patch("app.main.valid_google_url")
+@mock.patch("app.main.has_internet_connection")
+@pytest.mark.parametrize(
+    "internet_connection, google_url, result",
+    [
+        (False, False, "Not accessible"),
+        (False, True, "Not accessible"),
+        (True, False, "Not accessible"),
+        (True, True, "Accessible")
+    ]
+)
+def test_how_the_function_works(
+        mocked_connection: bool,
+        mocked_valid_url: bool,
```

Comment:
You should ensure that the parameter order in the test function definition matches the order that the decorators are applied